lib/install/node/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "webpacker-app",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Integrates Webpack to manage application-like JavaScript in Rails",
+  "main": "app/javascript/packs/index.js",
+
+  "scripts": {
+    "lint": "./node_modules/eslint/bin/eslint.js lib/ --fix",
+    "compile": "./bin/webpack",
+    "build": "NODE_ENV=production ./bin/webpack",
+    "server": "./bin/webpack-dev-server",
+    "watch": "./bin/webpack-watcher"
+  },
+
+  "_comment": "Customise as required, but do not remove otherwise your app will break",
+  "webpacker": {
+    "srcPath": "app/javascript",
+    "configPath": "config/webpack",
+    "nodeModulesPath": "node_modules",
+    "_comment": "If you rename distDir, please update distPath too",
+    "distDir": "packs",
+    "distPath": "public/packs",
+    "manifestFileName": "manifest.json"
+  },
+
+  "_comment": "Configure webpack-dev-server",
+  "devServer": {
+    "enabled": true,
+    "host": "localhost",
+    "port": "8080",
+    "compress": true
+  }
```

Comment:
Maybe default to 3500 rather than 8080? We've been doing that for the past 1.75 years for React on Rails and there have been no conflicts. I think 8080 is used sometimes.